hnn_core/tests/test_network.py

```diff
@@ -212,6 +212,40 @@ def test_network():
     net.clear_connectivity()
     assert len(net.connectivity) == 0
 
+    # Test LFP electrodes
+    kwargs_default = {
+        'electrode_pos': (2, 2, 400),
+        'sigma': 3.0, 'method': 'psa'}
```

Comment:
I wouldn't recommend using this pattern actually. I think it somehow made sense in the last test. For many newcomers to Python, kwargs is a confusing concept. If you can just set the variables like:

```py
electrode_pos = (2, 2, 400)
sigma = 3.0
method = 'psa'
```

also I think you need to have a new file `test_lfp.py` where you test the actual LFP calculations itself. Harder but must be done :)

Reply:
Maybe spend some time reading some of the tests in MNE-Python. It might help you to write better tests. For example:

https://github.com/mne-tools/mne-python/blob/main/mne/tests/test_epochs.py

You'll see how there is other stuff tested along with the `ValueError`